staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go

```diff
@@ -74,6 +74,26 @@ type ValidatingAdmissionPolicy struct {
 	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
 	// Specification of the desired behavior of the ValidatingAdmissionPolicy.
 	Spec ValidatingAdmissionPolicySpec `json:"spec,omitempty" protobuf:"bytes,2,opt,name=spec"`
+	// Status is the status of the ValidatingAdmissionPolicy, including warnings that are useful to determine if the policy
+	// behaves in the expected way.
+	// +optional
```

Comment:
Add details into API doc saying something like "users are not allowed to update this field"?

Reply:
It is clearer if we just say
```
// Populated by the system.
// Read-only.
```
adding